src/openlmis-reset-password/reset-password.html

```diff
@@ -18,19 +18,26 @@
                             {{'openlmisResetPassword.newPassword' | message}}
                         </label>
                         <input id="password"
-                            ng-model="vm.password"
-                            type="password"
-                            required/>
+                               ng-model="vm.password"
+                               ng-attr-type="{{ showPassword ? 'text' : 'password' }}"
+                               required/>
                     </div>
 
                     <div class="form-group">
                         <label for="reenteredPassword">
                             {{'openlmisResetPassword.reenterPassword' | message}}
                         </label>
                         <input id="reenteredPassword"
-                            ng-model="vm.reenteredPassword"
-                            type="password"
-                            required/>
+                               ng-model="vm.reenteredPassword"
+                               ng-attr-type="{{ showPassword ? 'text' : 'password' }}"
+                               required/>
+                    </div>
+
+                    <div class="form-group">
+                        <label>
+                            <input type="checkbox" id="showPassword" ng-model="showPassword" ng-checked="false">
```

Comment:
Add a test to check if the attribute changes.

Reply:
Done